dask_kubernetes/helm.py

```diff
@@ -121,7 +122,30 @@ async def _start(self):
         await super()._start()
 
     async def _get_scheduler_address(self):
-        service_name = f"{self.release_name}-{self.scheduler_name}"
+        # find the chart name, is there a better way?
+        chart = subprocess.run(
```

Comment:
Perhaps `subprocess.check_output` would be neater here?

Reply:
yup, that's better :)